google-cloud-storage/acceptance/storage/file_test.rb

```diff
@@ -316,6 +319,34 @@
     uploaded.delete
   end
 
+  it "should download, verify, and decompress a gzipped file with default md5 verification" do
+    skip "Not working. See https://github.com/GoogleCloudPlatform/google-cloud-ruby/issues/1835"
+
+    lazy_bucket = storage.bucket bucket_public_test_name
+    lazy_file = lazy_bucket.file file_public_test_gzip_name
+
+    Tempfile.open ["google-cloud", ".png"] do |tmpfile|
+      tmpfile.binmode
+      downloaded = lazy_file.download tmpfile
```

Comment:
Is md5 the default then? (Just trying to spot the differences between the tests.)

Reply:
Yes, md5 is the default.